src/store/Profile.ts

```diff
@@ -31,12 +31,14 @@ interface ProfileState {
   initialized: boolean
   currentProfile: ProfileModel
   isAuthenticated: boolean
+  isSettingVisible: boolean
```

Comment:
It does not seem necessary to add these methods to the store

Is there anything that prevents you to make the settings visibility state a state variable of `PageLayout`?

Reply:
I don't want to couple with Pagelayout

Pagelayout should be just for layout